feat: Implementate New supervisor page.

ClaaudioRocha opened the pull request.

ClaaudioRocha committed:
feat: Implementate New supervisor page.

yfernandes commented:
Hey, it's good practice to write a description when creating a pull request, in a small team with simple tasks the reviewer can still understand but that doesn't last long ok?!

yfernandes commented on apps/atqr/web/src/pages/ChooseNewSupervisor/ChooseNewSupervisor.html:
```diff
@@ -0,0 +1,90 @@
+    <link rel="stylesheet" href="../../assets\css\style.css">
+
+    <!-- SCSS -->
+    <link rel="stylesheet" href=".\ChooseNewSupervisor.scss"
```
This tag is missing it's ending bracket

yfernandes commented on apps/atqr/web/src/pages/ChooseNewSupervisor/ChooseNewSupervisor.html:
```diff
@@ -0,0 +1,90 @@
+    <link rel="stylesheet" href=".\ChooseNewSupervisor.scss"
+
+</head>
+        <!-- Contact Form Start -->
```
Where is the Body tag?

yfernandes commented on apps/atqr/web/src/pages/ChooseNewSupervisor/ChooseNewSupervisor.html:
```diff
@@ -0,0 +1,90 @@
+                                      <div class="col-md-6">
+                                          <!-- Single Form Start -->
+                                          <div class="single-form">
+                                              <input class="form-control" type="text" placeholder="Nome do Supervisor" id="supName">
```
Avoid abbreviations

yfernandes reviewed (commented):
Hi Rocha, your Html needs reworking and another review before merging. It's missing its body tag, which is a major component. However, in the time you weren't present we changed the frontend stack to use the pug pre-processor, so I think is best that you rework it in pug already. Is not very complicated, and the team can help you with this transition.

On another note, have you gotten to the js part of your journey? If so, you require you to implement one for this feature, if not I'll assign someone to do it for you. In any case, there is one more case that you have to take into consideration, if the user inputs an invalid email an error message should be presented to them, this message is also your responsibility.

The page is looking great keep studying!

ClaaudioRocha committed:
Fix tags in HTML and rename ids

Commit fixing tags in HTML and rename the ids with abreviations.

ClaaudioRocha committed:
feat: check inputs, validate e-mail and creation of .pug file

Creation of js file to verify inputs, validate the email adress of the new supervisor and creation of .pug file and transpile of the html to the file.

ClaaudioRocha committed:
feat: Invalid e-mail message

Creation of a invalid e-mail messsage.

ClaaudioRocha committed:
refactor web
